packages/core/src/util.ts

```diff
@@ -51,14 +51,63 @@ export function makeBacktrace(stack: string, shift = 0): BacktraceFrame[] {
           column: line.column
         }
       })
-    backtrace.splice(0, shift)
+    if (filterHbSourceCode) {
+      backtrace.splice(0, calculateBacktraceShift(backtrace))
+    }
+
     return backtrace
   } catch (_err) {
     // TODO: log error
     return []
   }
 }
 
+const DEFAULT_BACKTRACE_SHIFT = 3
```

Comment:
Are there other constants like this? Would be nice to store them all in one place, either at the top of the file or in a separate constants file.

Reply:
We have _some_ constants and so far we've been placing them near their usage (or where they most make sense).
For me, it works OK so far. But if you have a strong opinion on refactoring them (i.e. moving them to separate file, creating a convention), we can discuss it more 😄